docs/manual/src/swift/module.md

```diff
@@ -0,0 +1,34 @@
+# Compiling a Swift module
+
+Before you can import the generated Swift bindings as a module (say, to use them
+from your application, or to try them out using `swift` on the command-line) you
+first need to compile them into a Swift module.
+
+To do so, you'll need to crate a `.modulemap` file, which reassures Swift that it's
+safe to expose the underlying C FFI layer. It should look something like this,
+where `example` is the namespace as used in the `.udl` file:
+
+```
+module example {
+    header "path/to/uniffi_example-Bridging-Header.h"
+    export *
+}
+```
+
+Then use `swiftc` to combine the cdylib from your Rust crate with the generated
+Swift bindings:
+
+```
```

Comment:
This is based on what our tests do, and it felt weird not to have it documented anywhere.